build.xml

```diff
@@ -1683,6 +1683,11 @@ IMPORTANT: The '-Dpattern=' property is no longer a regular expression and is a
         <property name="goal8" value="-Djasig.ignore" />
         <property name="goal9" value="-Djasig.ignore" />
 
+        <condition property="threads" value="-T ${maven.threads.per.cpu}C">
```

Comment:
I received feedback that 1.5 was an optimal value from Oakland. It really depends on he server details.

Great idea! Using `else=` will make that section shorter and clear.